### PR TITLE
LF-17852: Check for size of container and modify height if not set

### PR DESCRIPTION
**Pull request by markdoten**
https://livefyre.atlassian.net/browse/LF-17852

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -69,8 +94,29 @@ MapComponent.prototype._initializeDOM = function(opts) {
...
+ * Poll the container element for it to resize, then resize the map relayout
+ * the map afterwards.
+ * @private
+ */
+MapComponent.prototype._pollForResize = function() {
+  var parentEl = this.containerEl.parent();
+  var self = this;
+
+  this._resizePoll = setInterval(function() {
+    var newHeight = parentEl.height();
```
Comment (markdoten) [inline src/main.js]: This is not good enough. What if it never has a size, but some ancestor does?

File: src/main.js [hunk 2]
```diff
@@ -69,8 +107,28 @@ MapComponent.prototype._initializeDOM = function(opts) {
   if (this.el.childNodes.length > 0) {
     return;
   }
+
   opts.el = document.createElement('div');
   this.el.appendChild(opts.el);
+  setTimeout($.proxy(this._handleSizing, this), 10);
```
Comment (ross-pfahler) [inline src/main.js]: is the timeout here to guarantee that things are in the DOM?

File: src/main.js [hunk 3]
```diff
@@ -69,8 +107,28 @@ MapComponent.prototype._initializeDOM = function(opts) {
...
+};
+
+/**
+ * Poll the container element for it to resize, then resize the map relayout
+ * the map afterwards.
+ * @private
+ */
+MapComponent.prototype._pollForResize = function() {
+  var parentEl = this.containerEl.parent();
```
Comment (ross-pfahler) [inline src/main.js]: do you think we should cache this so the lookup doesn't happen every 500ms?

File: src/main.js (hunk 2 above)
Comment (markdoten) [inline src/main.js]: Yes, this is created before app-embed has been injected into the DOM.

**Pull request merged**